config/default.json

```diff
@@ -73,6 +74,7 @@
     [ "redesignedSidebarBanner_20180222", "oldBanner" ],
     [ "businessPlanDescriptionAT_20170605", "original" ],
     [ "mobilePlansTablesOnSignup_20180330", "vertical" ],
-    [ "domainSuggestionKrakenV313_20180329", "group_0" ]
+    [ "domainSuggestionKrakenV313_20180329", "group_0" ],
+    [ "showMoneyBackGuarantee", "no" ]
```

Comment:
This needs to have the date specified to set the group correctly

Reply:
Yup, thanks for catching this